flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/failover/flip1/ExecutionFailureHandlerTest.java

```diff
@@ -59,17 +64,25 @@ class ExecutionFailureHandlerTest {
 
     private ExecutionFailureHandler executionFailureHandler;
 
+    private TrackingFailureEnricher trackingFailureEnricher;
```

Comment:
we should introduce a `TestingFailureEnricher`, because `TrackingFailureEnricher` already seems to be duplicated and we'd have to duplicate it in upcoming PRs as well

Reply:
Sure, done